apps/nextjs/src/app/dashboard/@breadcrumbs/container.tsx

```
@@ -0,0 +1,36 @@
+    "flex shrink-0 items-center gap-2",
+    "transition-[width,height] ease-linear",
+    "mx-4 mt-4 rounded-xl border-none bg-neutral-950",
+    LAYOUT_CONSTANTS.HEADER_HEIGHT,
+    `group-has-[data-collapsible=icon]/sidebar-wrapper:${LAYOUT_CONSTANTS.HEADER_HEIGHT_COLLAPSED}`,
+  ),
+  INNER: `flex items-center gap-2 px-4`,
+} as const;
+
+export function BreadCrumbsContainer({ children }: ChildrenProps) {
+  return (
+    <header role="banner" className={HEADER_CLASSES.CONTAINER}>
```

Biome (1.9.4)

[error] Interactive elements should not be assigned non-interactive roles.

WAI-ARIA roles should not be used to convert an interactive element to a non-interactive element.
Wrap your interactive element in a <div> with the desired role or put the content inside your interactive element.
Unsafe fix: Remove the role attribute.


(lint/a11y/noInteractiveElementToNoninteractiveRole)